azurerm/resource_arm_policy_assignment.go

```diff
@@ -146,6 +184,16 @@ func resourceArmPolicyAssignmentRead(d *schema.ResourceData, meta interface{}) e
 
 	d.Set("name", resp.Name)
 
+	if identity := resp.Identity; identity != nil {
+		d.Set("identity_type", resp.Identity.Type)
+		d.Set("identity_principal_id", resp.Identity.PrincipalID)
+		d.Set("identity_tenant_id", resp.Identity.TenantID)
+	}
+
+	if location := resp.Location; location != nil {
+		d.Set("location", location)
```

Comment:
(as above) can we normalize this prior to setting it in the state here by calling `azureRMNormalizeLocation`?

```suggestion
		d.Set("location", azureRMNormalizeLocation(*location))
```

Reply:
👍